examples/base/README.md

```diff
@@ -0,0 +1,26 @@
+# RainCatcher BaseModule
+
+This is a template for modules in the RainCatcher project, it contains the default dotfiles and other configuration files for the development tooling and the directory structure spected from individual modules.
+
+RainCatcher code is written in [TypeScript](http://typescriptlang.org), uses [Mocha](https://mochajs.org/) for unit tests and [Istanbul](https://istanbul.js.org/) for test coverage checks.
+
+For an explanation of the expected internals of see the [main README](../../README.md#package-structure)
+
+## Using this template
+
+When creating a new module, copy the entire contents of this folder and make the following adjustments to [`package.json` ](./package.json):
+
+  - `name`: Update to new package's name
+  - `version`: Set it to your new intended semver (i.e. `0.0.0` or `1.0.0`)
+  - `description`: Remove the description that contains
```

Comment:
`Remove the description that contains` I'm not really sure what this means. Is this sentence incomplete? or does this mean 'remove the existing description and replace it with your own'?